models/Element/Traits/ScheduledTasksTrait.php

```diff
@@ -59,20 +61,18 @@ public function setScheduledTasks($scheduledTasks)
         return $this;
     }
 
-    public function saveScheduledTasks()
+    public function saveScheduledTasks(): void
```

Comment:
```suggestion
    public function saveScheduledTasks()
```
Adding the return type is a BC break.